src/core/components/site-list/site-list.ts

```diff
@@ -0,0 +1,80 @@
+// (C) Copyright 2015 Moodle Pty Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Component, ContentChild, Input, Output, TemplateRef, EventEmitter } from '@angular/core';
+
+import { CoreSiteBasicInfo } from '@services/sites';
+import { CoreAccountsList } from '@features/login/services/login-helper';
+
+/**
+ * Component to display a list of sites (accounts).
+ *
+ * By default this component will display the avatar and user fullname for each site, but it allows adding more information
+ * in the item and in the label for each site, using #siteItem and #siteLabel ng-templates. These templates will receive the
+ * site being rendered and whether it's the current site or not. Example:
+ *
+ * <core-site-list [accountsList]="accountsList">
+ *     <ng-template #siteLabel let-site="site" let-isCurrentSite="isCurrentSite">
+ *         <!-- Content to be placed in the label, after the user full name.
+ *     </ng-template>
+ *
+ *    <ng-template #siteItem let-site="site" let-isCurrentSite="isCurrentSite">
+ *         <!-- Content to be placed in the item.
+ *     </ng-template>
+ * </core-site-list>
+ */
+@Component({
+    selector: 'core-site-list',
+    templateUrl: 'site-list.html',
+    styleUrls: ['site-list.scss'],
+})
+export class CoreSiteListComponent<T extends CoreSiteBasicInfo> {
```

Comment:
```suggestion
export class CoreSitesListComponent<T extends CoreSiteBasicInfo> {
```

I'm aware that this change requires modifying a lot of code and filenames, sorry :sweat_smile:.

Reply:
I renamed all the files and CSS classes, I think I didn't miss any.